### ISSUE TITLE
Querying number of threads and/or processes used from dask.config

### ISSUE THREAD

**Issue by jakirkham** [dask/dask#3608]
Would be nice if there was a way to inspect `dask.config` and get the number of threads being used and the number of processes used (likely as to different values). The latter is particularly helpful when trying to determine if pickling comes into the picture.

**Comment by mrocklin** [comment]
I agree that this would be a useful thing.  I don't have a good understanding of how to implement it reliably across all scheduler.

In either case, I think that this is possibly outside of the scope of the dask.config module. (though that's a conversation for after someone actually finds a way to reliably implement this)

**Comment by jakirkham** [comment]
Is it possible to have schedulers derive from a base class with some abstract methods for these checks or would that be too difficult?

**Comment by mrocklin** [comment]
It's worth investigating.  This is part of a much larger question though of
how to define requirements for a scheduler in the future.  Currently all we
demand is the get function.
